### PR TITLE
Update claim_greater.txt

### PR DESCRIPTION
**Pull request by DoktorKure**
fixed legacy decision requirements by adding in democracy1 and hms_government1

### CODE REVIEW

**Comment by ccfox** [comment]
I guess they were missed in the automated find/replaces I ran due to the formatting.

File: CWE/decisions/claim_greater.txt [hunk 1]
```diff
@@ -173,7 +177,7 @@ political_decisions = {
 				ruling_party_ideology = populist
 				ruling_party_ideology = nationalist
 				jingoism = 24
-NOT = { government = democracy government = hms_government }
+				NOT = { government = democracy government = democracy1 government = hms_government government = hms_government1 }
```
Comment (settintotrieste) [inline CWE/decisions/claim_greater.txt:180]: Should be NOT = { OR = { government = democracy government = democracy1 government = hms_government government = hms_government1 } }

**Comment by DoktorKure** [comment]
Should be good now

File: CWE/decisions/claim_greater.txt (hunk 1 above)
Comment (settintotrieste) [reply CWE/decisions/claim_greater.txt]: Thanks!

**Pull request merged**